[WIP] API docs contrib guide (#1592)
elastic/docs-builder: docs/contribute/api-docs-contribution/organize-annotate.md

**Add new tags**

If you create a new tag value, you must also add it to the [elasticsearch-shared-overlays.yaml](https://github.com/elastic/elasticsearch-specification/blob/main/docs/overlays/elasticsearch-shared-overlays.yaml) file. You can see all existing tag values in that file.

Use **sentence case** for tag names. If you need different casing in the final docs, use the `x-displayName` extension in the overlay file.

:::{note}
Each operation can only have one tag, even though the OpenAPI specification supports multiple tags. This is a deliberate system constraint to keep the docs navigation clean and predictable.
:::
:::

:::::

## Extend OpenAPI documents

Per [OpenAPI Specification v3.0.3](https://spec.openapis.org/oas/v3.0.3#specification-extensions):

_"While the OpenAPI Specification tries to accommodate most use cases, additional data can be added to extend the specification at certain points."_

The extensions properties are implemented as patterned fields that are always prefixed by `x-`. The following extensions are supported by Bump.sh:

| **Field**       | **Description** |
|-----------------|-----------------|

> | `x-beta` | Indicates that something is in beta. **Note:** Prefer `x-state` instead.<br> <br>Content with `x-beta` is **not** included in Bump.sh’s automatic changelog. [More info](https://docs.bump.sh/help/specification-support/doc-beta/) |

Reviewer: Since we've turned off the changelogs, it's likely better to omit mention of them.
```suggestion
| `x-beta` | Indicates that something is in beta. **Note:** Use `x-state` instead. |
```
